### ISSUE TITLE
injectBaseStyles cannot be false

### ISSUE THREAD

**Issue by andrejcremoznik** [imgix/luminous#105]
It's impossible to disable base style injection because any falsey value will always result in `true`.
```
const injectBaseStyles = options["injectBaseStyles"] || true;
```
[src/js/Luminous.js#70](https://github.com/imgix/luminous/blob/bbec5179c839feaa4756233b21cbd60ba8bfb81c/src/js/Luminous.js#L70)

I'd do a one line fix but I think the entire options object block should be refactored to something more modern-like.

E.g.

```
// ES2015 Object.assign
this.settings = Object.assign({}, defaults, options)

// OR object spread
this.settings = { ...defaults, ...options }

// Or some day in the far future
const injectBaseStyles = options["injectBaseStyles"] ?? true;
```

**Comment by Lefthandmedia** [comment]
this is still an issue today. the fix did not end up in Master so it seems.

**Comment by sherwinski** [comment]
Hi @andrejcremoznik @Lefthandmedia, sorry for the extreme delay but the fix for this issue is now released in [v2.3.3](https://www.npmjs.com/package/luminous-lightbox) 🎉
